### ISSUE TITLE
 KeePassXC-Browser has encountered an error

### ISSUE THREAD

**Issue by KitchM** [keepassxreboot/keepassxc#1606]
When selecting icon on address bar, the following is displayed:
"KeePassXC-Browser has encountered an error:
Timeout or not connected to KeePassXC"

No fix is able to be made in settings.

KeePassXC - Version 2.3.0
Revision: 4c0ed74

Libraries:
- Qt 5.2.1
- libgcrypt 1.8.1

Enabled extensions:
- Auto-Type
- Legacy Browser Integration (KeePassHTTP)
- SSH Agent
- YubiKey
 
Operating system: Xubuntu 14.04
CPU architecture: AMD 64-bit
Kernel: Latest

Enabled extensions:
AdBlocker for YouTube™	0.2.8
KeePassXC-Browser	1.0.0

**Comment by droidmonkey** [comment]
This would lead me to believe that your KeePassXC is not running or you haven't fully enabled the browser integration.

**Comment by dCo3lh0** [comment]
maybe i can add my experience too here, im on Linux Mint 18.2 XFCE which is based on 16.04, im running KeePassXC completely fine but can't synchronize with my browser which is Waterfox 56.0.3 i have the KeePassXC-Browser installed, tried selecting the credential fields, everything works fine, but can't connect to KeePass Database, does this have anything to do with a browser which is not included but based on the browser options found on KeePass?

**Comment by Generator** [comment]
Follow the instructions here
[How to connect KeePassXC-Browser with KeePassXC](https://keepassxc.org/docs/keepassxc-browser-migration/)

**Comment by KitchM** [comment]
Okay, I'll contact them.  Thanks.

**Comment by KitchM** [comment]
By the way, how do we get in touch with the KeePassXC packager?

**Comment by Generator** [comment]
@KitchM https://launchpad.net/ubuntu/+source/keepassxc/+bugs
"Report a bug"

**Comment by TheZ3ro** [comment]
Wait a second, you have Xubuntu 14.04 and you are using Qt 5.2.1.
KeePassXC-Browser needs Qt5.3 and so It's not supported on (X)Ubuntu 14.04.
Please use the AppImage.

Duplicate of #1642

**Comment by KitchM** [comment]
What is AppImage?

**Comment by KitchM** [comment]
Okay, I found the appimage version and downloaded it.  I made it executable.  I changed the shortcut to use it.  I opened it and it worked fine.

Next I opened Firefox Quantum Version 59.0 and got the same message.  I removed the add-on.  Then I searched for the current add-on and installed it.  Then I tried again and got the same message: "KeePassXC-Browser has encountered an error: Timeout or not connected to KeePassXC"

I then opened Settings, and selected the Connected Databases tab.  There were none found, so I selected to Connect hoping that it would allow me to search for the correct one, but that doesn't work either.

One would assume that the database the is recognized by the main program (and is open) would be seen by the browser add-on.
